app/managers.py

```diff
@@ -0,0 +1,59 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._conection = sqlite3.connect("../cinema")

```

Comment:
Create a connection assuming that db is in the current working directory and has a `db3` extension.